### PR TITLE
Update 1st-level rollable tables

### PR DESCRIPTION
**Pull request by joeparis**
Refresh 1st-Level Consumable Items and 1st-Level Permanent Items tables with remaster changes.

### CODE REVIEW

File: packs/rollable-tables/1st-level-consumables.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "_id": "tlX5PLwar8b1tmiQ",
-    "description": "Table of 1st-Level Consumables",
+    "_id": "ymDhLpTQPaZLqc7O",
```
Comment (simonward) [inline packs/rollable-tables/1st-level-consumables.json:2]: We might want to preserve the ids?

Comment (joeparis) [reply packs/rollable-tables/1st-level-consumables.json:2]: Sorry, I created new tables without thinking about the fact that the old IDs should be reserved. I apologize, I should have thought about that more than I did.

Comment (joeparis) [reply packs/rollable-tables/1st-level-consumables.json]: Should be good to go.

File: packs/rollable-tables/1st-level-consumables.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "_id": "tlX5PLwar8b1tmiQ",
-    "description": "Table of 1st-Level Consumables",
+    "description": "",
```
Comment (simonward) [inline packs/rollable-tables/1st-level-consumables.json:3]:
```suggestion
    "description": "Table of 1st-Level Consumables",
```

Comment (joeparis) [reply packs/rollable-tables/1st-level-consumables.json]: Thanks for your patience and help. I'll get it right next time.

**Pull request closed without merging**